src/css/Main.css

```diff
@@ -6,6 +6,7 @@
 	align-items: flex-start;
 	align-content: flex-start;
 	margin-top: 80px;
+	padding-top:7px;
```

Comment:
This is such a small detail but makes that error page look so much better, thank you so much for adding that in!